README.md

```diff
@@ -8,49 +8,37 @@ https://cdsvacpoc.azurewebsites.net/
 
 The setup documentation can be found below. If you'd like to contribute to the project, we have more detailed documentation regarding our tech choices here: [DOCUMENTATION.md](./DOCUMENTATION.md).
 
-## Quickstart
-
-- `yarn install`
-- `yarn dev`
-
-## How to get VAC Benefits Directory to work on VAC machines (May 31, 2018)
-
-- download Node.JS of version 9 or newer(as .zip since .msi is blocked)
-
-- following steps required tp set up the newly installed Node cmd line commands such as npm globally
-
-  > Start menu > Control Panel > User accounts > user accounts > change my environment variables >
-  > select path > Edit > Now enter the location of where you have Node installed with a ; to seperate
-  > between any other entries here.
-  > EX: C:\dev ; C:\Node
-
-- `npm install -g yarn`
-- `yarn install` (While in project main directory, takes a while, especially on building packages)
-
-- `yarn dev` (run while CD'd in project folder used to start the local server)
-
-- If you run into syntax errors chances are there are missing packages/dependencies and you may want to try
-  running a clean yarn install. Delete Node_modules folder and then re run `yarn install` in the main folder.
-
-## Configuration
-
-The application uses Sentry.io for error logging. If you would also like to use sentry, make sure your public client key is the `SENTRY_DSN` environment variable.
-
-Google Analytics is used for behaviour tracking, ex: How many people switch between languages. If you would like to include your own GA key set the `GA_UA` environment variable.
-
-The Google maps api is used to display a map of area office locations. If you would like to include your own api key set the `GOOGLE_MAPS_KEY` environment variable.
-
-Here is an example of how to add an ENV variable locally on OSX:
+## Environment variables
+
+These need to be set on the production service for the app to function correctly.
+Some are also required for testing. You will also need some of these set for local development
+(at the very least, you should have `AIRTABLE_READ_KEY` and `GOOGLE_MAPS_KEY`).
+Contact other developers on the project for what values we're currently using.
+
+| Variable                     | Use                                                     | Where                           |
+| ---------------------------- | ------------------------------------------------------- | ------------------------------- |
+| `AIRTABLE_READ_KEY`          | load data (benefits / translations / etc) from Airtable | production / locally            |
```

Comment:
```
import random

keys = [AIRTABLE_READ_KEY,AIRTABLE_WRITE_KEY,BROWSERSTACK_ACCESS_KEY,BROWSERSTACK_USERNAME,GA_UA,GITHUB_PUBLIC_ACCESS_TOKEN,GOOGLE_MAPS_KEY,SENTRY_DSN]

print random.shuffle(keys)
```

Reply:
I R confuzd 🙃

Reply:
You changed the key order from alphabetical to random 😈

Reply:
not random - by importance :)